test/unit/renderer/viz/expressions/ramp.test.js

```diff
@@ -1,5 +1,11 @@
-import { validateStaticType, validateStaticTypeErrors, validateDynamicTypeErrors } from './utils';
-import { ramp, buckets } from '../../../../../src/renderer/viz/expressions';
+import { validateStaticType, validateStaticTypeErrors, validateDynamicTypeErrors, checkRGBAThreshold } from './utils';
+import * as cartocolor from 'cartocolor';
+import { ramp, buckets, palettes, globalQuantiles, linear } from '../../../../../src/renderer/viz/expressions';
+import * as s from '../../../../../src/renderer/viz/expressions';
+import { hexToRgb } from '../../../../../src/renderer/viz/expressions/utils';
+import Metadata from '../../../../../src/renderer/Metadata';
+
+const DEFAULT_COLOR = s.namedColor('gray');
```

Comment:
Couldn't we get the default color from the ramp? It could be interesting for end-users to be able to do that to make legends.

Reply:
I think we should address that in a different issue.

Reply:
:+1: